tools/cldr-apps/src/main/java/org/unicode/cldr/web/STFactory.java

```diff
@@ -1172,7 +1174,11 @@ public synchronized void voteForValue(User user, String distinguishingXpath, Str
                 doPermanentVote(distinguishingXpath, xpathId, value);
             }
 
-            xmlsource.setValueFromResolver(distinguishingXpath, null, false /* resolveMorePaths */);
```

Comment:
This is an opportunity for changing the boolean into something meaningful, 

add in XmlSource
enum PathResolution {MORE_PATHS, FEWER_PATHS}

true => PathResolution.MORE_PATHS

But probable give an more descriptive enum value name than MORE; that is what is included that isn't in LESS.

Reply:
yes, and, do all callers now pass true ?If so, then just drop the parameter

Reply:
loadVoteValues still calls setValueFromResolver with resolveMorePaths true for vxml generation, and false otherwise. Before we could change that, we would need to study and measure the effects, including possible performance costs. I think that's beyond the scope of this ticket, which is for fixing a well-defined bug.

For loadVoteValues, this comment describes resolveMorePaths:

         * @param resolveMorePaths false to do vote resolution only on paths with votes in current votes table, or
         *                         true to do vote resolution also on all paths in trunk (for making vxml).

Should we make another ticket with the objective of cleaning up these parts of the code? I hesitate to create another techdebt ticket when we already have so many.